modin/error_message.py

```diff
@@ -21,10 +21,10 @@ class ErrorMessage(object):
     printed_warnings = set()
 
     @classmethod
-    @logger_decorator("MODIN-ERROR", "ErrorMessage.not_implemented", "debug")
     def not_implemented(cls, message=""):
         if message == "":
             message = "This functionality is not yet available in Modin."
+        get_logger().info(f"Modin Error: NotImplementedError: {message}")
```

Comment:
Is this useful given that the message may very well be `This functionality is not yet available in Modin.`? Maybe we should only print the message if its not empty, because then it may be that the lower level is trying to surface something?

Reply:
I think it's fine to still log this even if the message is `This functionality is not yet available in Modin. `. I think just logging that this error was raised in the logs is useful.

Reply:
Sgtm!